releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -2,6 +2,9 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.1 - January, 21 2020
+  * Supported [Azure Active Directory authentication](https://github.com/Azure/AppConfiguration/blob/master/docs/REST/authorization/aad.md) of fetching key-values from App Configuration instance.
```

Comment:
This is a breaking change. I think we should provide more details to tell users how to migrate from what it was to what it is now (for example, before/after).

Reply:
Updated, please take a look